Pipfile

```diff
@@ -0,0 +1,137 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+ckanapi = "==4.3"
+alembic = "==1.0.0"
+babel = "==2.7.0"
+beaker = "==1.11.0"
+bleach = "==3.1.4"
+certifi = "==2021.10.8"
+chardet = "==4.0.0"
+click = "==8.1.2"
+dominate = "==2.4.0"
+fanstatic = "==1.1"
+feedgen = "==0.9.0"
+flask-babel = "==1.0.0"
+flask-multistatic = "==1.0"
+flask = "==1.1.1"
+funcsigs = "==1.0.2"
+idna = "==3.3"
+itsdangerous = "==1.1.0"
+jinja2 = "==2.11"
+lxml = ">=2.3"
+mako = "==1.1.5"
+markdown = "==2.6.7"
+markupsafe = "==1.1.1"
+nose = "==1.3.7"
+passlib = "==1.6.5"
+polib = "==1.0.7"
+psycopg2 = "==2.8.2"
+PyJWT = {version = "==2.1.0",extras = ["crypto"]}
+pysolr = "==3.6.0"
+python-dateutil = "==2.8.2"
+python-editor = "==1.0.4"
+python-magic = "==0.4.15"
+# Because of Pandas dependencies
```

Comment:
is that for pytz? Could you give a little more info on why exactly Pandas needs it? Maybe an issue link etc.
It would help a lot in the future when hopefully we'll be able to get rid fo this dependency freeze :)